starter-packs/refinery-deployment/cloudformation-aws/image-builder/configure-refinery.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash -x
+
```

Comment:
```suggestion
set -exo pipefail
```

https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425

Reply:
Thank you for the suggestion!  I've actually designed this script to have explicit error handling at every step intentionally, so as to make redundant (but also interact dangerously) with `set -e -o pipefail`.  For example"

```
test -n "$REDIS_HOST" || exit 1
test -n "$REDIS_PORT" || exit 1
test -n "$REDIS_PASSWORD" || exit 1
test -n "$HONEYCOMB_API_KEY" || exit 1
```

For this type of script I think the explicit error handling is more appropriate.  and regarding the -x it is useful here, and cfn-init when it runs magically adds that in somehow.